src/api.ts

```diff
@@ -1,19 +1,58 @@
+// @ts-nocheck There are some issues in the Definitely Typed packages of the "ref" related dependencies.
 import { INCHIAPI } from "./ffis";
+import { inchi_OutputStruct, inchi_OutputStructEx } from "./headers";
 
-export function CheckINCHIKey(input: string): 0 | -1 | 1 | 2 | 3 {
-  return INCHIAPI.CheckINCHIKey(input) as 0 | -1 | 1 | 2 | 3;
-}
+// #region Types and Interfaces
 
-export function CheckINCHI(input: string, strict?: boolean): 0 | -1 | 1 | 2 | 3 | 4 {
-  return INCHIAPI.CheckINCHI(input, strict === true ? 1 : 0) as 0 | -1 | 1 | 2 | 3 | 4;
-}
+export type StatusReturnType1 = -1 | 0 | 1 | 2 | 3;
+export type StatusReturnType2 = -1 | 0 | 1 | 2 | 3 | 4;
 
-export function GetStringLength(input: string): number {
-  return INCHIAPI.GetStringLength(input);
-}
+export const GetINCHIReturnCode = {
+  /**
+   * Success; no errors or warnings
+   */
+  inchi_Ret_OKAY: 0,
+  /**
+   * Success; warning(s) issued
+   */
+  inchi_Ret_WARNING: 1,
+  /**
+   * Error: no InChI has been created
```

Comment:
@qur786 Similar to this object, create objects for `StatusReturnType1` and `StatusReturnType2` too.